contracts/bond/DaoBondConfiguration.sol

```diff
@@ -49,6 +49,9 @@ abstract contract DaoBondConfiguration is DaoBondCollateralWhitelist {
 
         _setTreasury(_daoConfigLastId, erc20CapableTreasury);
 
+        DaoBondConfig storage config = _daoConfig[_daoConfigLastId];
```

Comment:
issue (blocking) : remove these lines.

Duplication of the logic in the function call above `_setTreasury()`